corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -50,7 +51,8 @@
 # cf. http://www.theguardian.com/environment/2016/apr/17/boaty-mcboatface-wins-poll-to-name-polar-research-vessel
 
 
-class CaseSearchTests(TestCase, ElasticTestMixin):
+@attr(es_test=True)
+class CaseSearchTests(ElasticTestMixin, TestCase):
```

Comment:
Is the mixin change a bugfix?

Reply:
It may not have been this particular test, but some other test failed, so I swapped others as well. There was some initiation issue.